docs/Gopkg.toml.md

```diff
@@ -14,6 +14,15 @@ required = ["github.com/user/thing/cmd/thing"]
 * Aren't `import`ed by your project, [directly or transitively](FAQ.md#what-is-a-direct-or-transitive-dependency)
 * You don't want put in your `GOPATH`, and/or you want to lock the version
 
+Please note that this only pulls in the sources of these dependencies. It does not install or compile them. So, if you need the tool to be installed you should still run the following (manually or from a `Makefile`)  after each `dep ensure`:
+
+```bash
+cd vendor/pkg/to/install
```

Comment:
i meant...something else, because i have apparently been super wrong about how this works for a very, very long time 😮 😱 

i've never been happier to be wrong, though. being able to compile correctly from within `vendor` is a huge boon, and something that i thought was basically a lost cause.